### PR TITLE
Refactor main

### PR DESCRIPTION
**Pull request by fermadeiral**

### CODE REVIEW

File: src/main/java/sonarquberepair/Main.java [hunk 1]
```diff
@@ -8,39 +8,22 @@
...
 
-	public static void repair(String pathToFile, String projectKey, int ruleKey) throws Exception {
-		Launcher launcher = new Launcher(){};
-		launcher.getEnvironment().setPrettyPrinterCreator(() -> {
-					SniperJavaPrettyPrinter sniper = new SniperJavaPrettyPrinter(launcher.getEnvironment());
-					sniper.setIgnoreImplicit(false);
-					return sniper;
-				}
-		);
+	public static void repair(String pathToFile, String projectKey, int ruleKey, boolean sniperMode) throws Exception {
```
Comment (monperrus) [inline src/main/java/sonarquberepair/Main.java]: prefer an enum over a boolean: 1) it gives more readable code 2) it allows for backward compatible extensions in the future

```
enum PrettyPrintingStrategy {
  NORMAL,
  SNIPER,
}
```

**Comment by henry-lp** [comment]
@fermadeiral sorry I did not know that someone already working on a refactor :) . Perhaps we can merge two PR together somehow (my #64) . WDYT ? :)

File: src/main/java/sonarquberepair/Main.java (hunk 1 above)
Comment (fermadeiral) [reply src/main/java/sonarquberepair/Main.java]: Great! Done.

**Comment by fermadeiral** [comment]
> @fermadeiral sorry I did not know that someone already working on a refactor . Perhaps we can merge two PRs together later (my #64) . WDYT ? :)

Henry, first of all, I've worked to only organize this repository, before releasing it, so then we can have a nice ground to actually start making the tool better, to add new features, etc. For that reason, I need that this PR is merged soon, I cannot wait for your PR. Moreover, remember, let's always do smallest PR as possible, only handling one type of issue per time, otherwise it's difficult to review changes.

**Comment by henry-lp** [comment]
> 
> 
> > @fermadeiral sorry I did not know that someone already working on a refactor . Perhaps we can merge two PRs together later (my #64) . WDYT ? :)
> 
> Henry, first of all, I've worked to only organize this repository, before releasing it, so then we can have a nice ground to actually start making the tool better, to add new features, etc. For that reason, I need that this PR is merged soon, I cannot wait for your PR. Moreover, remember, let's always do smallest PR as possible, only handling one type of issue per time, otherwise it's difficult to review changes.

Right, I understand. I mean can rebase my PR after your merge :) . It's no problem .

**Comment by monperrus** [comment]
Thanks Fernanda!

**Pull request merged**